cms/test_utils/project/templates/registration/logged_out.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+
+{% block content %}
+
+<p>{% trans "Thanks for spending some quality time with the Web site today." %}</p>
```

Comment:
This is just the standard django logout message 
This template it's just for testing purposes, it's never shown to the user